README.md

```
@@ -29,4 +29,120 @@ public class Test{
+        Printer evenPrinter = new Printer(counter, PrinterType.EVEN);
+
+        Thread oddPrinterThread = new Thread(oddPrinter, "Odd-Printer");
+        Thread evenPrinterThread = new Thread(evenPrinter, "Even-Printer");
+
+        oddPrinterThread.start();
+        evenPrinterThread.start();
+    }
+}
+```
+
+> You can find the source code in the [com.anusruta.multithreading.printoddeven]()
```

markdownlint-cli2 (0.18.1)

No empty links

(MD042, no-empty-links)